recipes/file/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
This line is not necessary, ``conda-build`` already do that
```suggestion
```

Reply:
Nice, it's good to know. And I feel it takes care of the permissions (i.e., `chmod 755 build.sh`) as I didn't have to set the permissions on my side.

Have people ever consider to run such scripts beyond the conda-build approach (such as building a package using the conda-build's `build.sh` script)?

Reply:
Not that I am aware of

But basically, the steps on ``build.sh`` are those necessary to compile the project. To create the package, solve dependencies, install dependencies, set environment variables, set ``c++`` flags, and so on... ``Conda`` takes care for you

Reply:
After removing that shebang I think it is good to go

Reply:
Sounds good. Indeed, the `build.sh` script may not be 100% reusable outside of the conda ecosystem as it may contain some conda-specific vars like prefixes, `$PYTHON` and so on.